### PR TITLE
Adds create call twice by same authorizer test

### PR DESCRIPTION
**Pull request by jeremy-then**
Add create already voted test

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java [hunk 1]
```diff
@@ -2270,6 +2270,28 @@ void voteFederationChange_create_returnsSuccessfulResponseCode() {
 
         }
 
+        @Test
+        void voteFederationChange_whenCreateCallTwiceBySameAuthorizer_returnsGenericResponseCode() {
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2274]: __returnsGenericErrorResponseCode_

Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2274]: __whenCreateIsCalledTwiceBySameAuthorizer_

**Review by marcos-iov — APPROVED** [review]
Looks like we are missing a test where we call create with 2 different authorizers, then assert that a new pending federation was created.

Unless that test is somewhere else, please a create a task to do it

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java (hunk 1 above)
Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java]: Updated.

**Review by julia-zack — APPROVED** [review]

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java [hunk 2]
```diff
@@ -2282,6 +2282,28 @@ void voteFederationChange_create_returnsSuccessfulResponseCode() {
...
+
+            // Arrange
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            ABICallSpec abiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Act
+
+            // First create call
+            federationSupport.voteFederationChange(tx, abiCallSpec, signatureCache, bridgeEventLogger);
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2296]: We should assert the first call was successful

Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java:2296]: I was following previous suggestions, about avoiding to test what has been already tested. The test exactly right before this up to this point is exactly the same and tested this. But it won't hurt to assert it.

Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java]: Updated.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**